### PR TITLE
feat: new seed phrase

### PR DESCRIPTION
**Pull request by angelia-yuqi-personal**
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/104)

### CODE REVIEW

**Review by yqrashawn — CHANGES_REQUESTED** [review]

**Review by RickStormBringer — APPROVED** [review]

File: packages/popup/src/pages/CreateSeed/BackupSeed.js [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+import useGlobalStore from '../../stores'
+import {SeedWord} from './components'
+
 function BackupSeed() {
-  return <div>BackupSeed</div>
+  const {t} = useTranslation()
+  const history = useHistory()
+  const {setCreatedMnemonic} = useGlobalStore()
+  const [mnemonic, setMnemonic] = useState([])
+  const {data: mnemonicData} = useRPC(['wallet_generateMnemonic'], undefined, {
```
[the remark this answers is not in the document]
Comment (angelia-yuqi-personal) [reply packages/popup/src/pages/CreateSeed/BackupSeed.js]: done

File: packages/popup/src/pages/CreateSeed/ConfirmSeed.js [hunk 2]
```diff
@@ -1,5 +1,113 @@
...
+  const onCreate = () => {
+    setImportingMnemonic(true)
+    if (mnemonic !== createdMnemonic) {
+      setMnemonicError(t('confirmSeedError'))
+      return
+    }
+    request('wallet_importMnemonic', {
+      mnemonic,
+      password: createdPassword,
+      force: true,
```
[the remark this answers is not in the document]
Comment (angelia-yuqi-personal) [reply packages/popup/src/pages/CreateSeed/ConfirmSeed.js]: remove the force create now

File: packages/popup/src/pages/CreateSeed/ConfirmSeed.js [hunk 3]
```diff
@@ -1,5 +1,113 @@
...
+    mnemonicArray.splice(index, 1, word)
+    setMnemonic(mnemonicArray.join(' '))
+  }
+  const getDisabled = word => {
+    const mnemonicArray = mnemonic.split(' ')
+    const index = mnemonicArray.findIndex(item => word === item)
+    return index > -1
+  }
+  const onCreate = () => {
+    setImportingMnemonic(true)
```
[the remark this answers is not in the document]
Comment (angelia-yuqi-personal) [reply packages/popup/src/pages/CreateSeed/ConfirmSeed.js]: done

File: packages/popup/src/pages/CreateSeed/ConfirmSeed.js [hunk 4]
```diff
@@ -1,5 +1,113 @@
...
+    if (mnemonic !== createdMnemonic) {
+      setMnemonicError(t('confirmSeedError'))
```
[the remark this answers is not in the document]
Comment (angelia-yuqi-personal) [reply packages/popup/src/pages/CreateSeed/ConfirmSeed.js:38]: done

**Review by yqrashawn — APPROVED** [review]

**Pull request merged**